### ISSUE TITLE
Does it need to be valid during training?

### ISSUE THREAD

**Issue by Ijustakid** [tensorflow/models#8431]
I used train_image_classifier.py to fine-turning mobilenet v2 models, but I found that there is no verification step in the training process.
Is that true? or am I missing something？

PS: Should I use eval_image_classifier.py after end of training?

Thanks！
br

**Comment by Ijustakid** [comment]
And, accuracy will not be displayed during training. How do I know the performance of the current model parameters? Can I only check by the loss value during training??
